charts/barman/values.yaml

```diff
@@ -32,6 +32,20 @@ deployment:
   imagePullSecrets: []
   # -- Specify additional ENVs for the Deployment
   additionalENVs: {}
+  # Init containers to add to deployment's pod spec. At least one plugin provider image is required.
+  # If the value is a string then it is evaluated as a template.
```

Comment:
```suggestion
  # If the value is a string then it is evaluated as a template.
  # -- Specify initContainers for the deployment
```

Reply:
We need this to generate the README.md you modified manually.

Reply:
New to helm docs
done (I guess)